integration_tests/features/WalletCli.feature

```diff
@@ -0,0 +1,115 @@
+Feature: Wallet CLI
+
+    Scenario: As a user I want to change base node for a wallet via command
+        Given I have a base node NODE1 connected to all seed nodes
+        And I have a base node NODE2 connected to all seed nodes
+        And I have wallet WALLET connected to base node NODE1
+        Then I change base node of WALLET to NODE2
+
+    Scenario: As a user I want to set and clear custom base node for a wallet via command
+        Given I have a base node NODE1
+        And I have a base node NODE2
+        And I have wallet WALLET connected to base node NODE1
+        Then I set custom base node of WALLET to NODE2
+        And I clear custom base node of wallet WALLET
+
+    Scenario: As a user I want to change password via command
+        Given I have wallet WALLET connected to all seed nodes
+        When I stop wallet WALLET
+        And I change the password of wallet WALLET to changedpwd
+        Then the password of wallet WALLET is not kensentme
+        Then the password of wallet WALLET is changedpwd
+
+    Scenario: As a user I want to get balance via command
+        Given I have a base node BASE
+        And I have wallet WALLET connected to base node BASE
+        And I have mining node MINE connected to base node BASE and wallet WALLET
+        And mining node MINE mines 5 blocks
+        Then I wait for wallet WALLET to have at least 1000000 uT
+        And I stop wallet WALLET
```

Comment:
See comment below

Reply:
It's above in my view of the PR :-) I don't know why the order is reversed... anyway, the same applies.